recipes/asyncinotify/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "asyncinotify" %}
+{% set version = "2.0.2" %}
+{% set blakesha256 = "9dc15e1af41dc171fbfdd951ecdc773ead5d431ff88df57bf4e23467a5bf4407" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/{{ blakesha256[0:2] }}/{{ blakesha256[2:4] }}/{{ blakesha256[4:] }}/{{ name }}-{{ version }}.tar.gz
```

There's a URL that only depends on the package name rather than the hash:
```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```